### ISSUE TITLE
Latest Posts: When the full content option is used, videos are not contained and shows with the original width and height

### ISSUE THREAD

**Issue by carolinan** [WordPress/gutenberg#61477]
### Description

In the latest posts block, when the option "Post content > Show full post" is used, videos have no width set on the front of the site. So they are not contained to the space that is allocated for the post. It expands outside the list item and covers / overlaps other content and can create a horizontal scroll bar.

For example in my test, I set the block to use a five column grid.
In the editor, the video was 107px wide and was contained to its column.
On the front, the video was **3016px** wide, the media file's original width.

### Step-by-step reproduction instructions

Create a new post and add a video block, or: a cover block or media & text block with a video.

Add a latests posts block (In the same post, a new post or page, or in a template, it does not matter).
Enable "Post content > Show full post" in the block settings panel of the latests post block.
Confirm that the video(s) are reasonably sized and does not overlap any other content in the editor.
Save and view the front of the site.
Note the size of the video.

Test both the grid layout with different columns and the default list view layout.

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by amitraj2203** [comment]
Hi @carolinan I have raised a [PR](https://github.com/WordPress/gutenberg/pull/61538) for its fix.

Thanks.

**Comment by carolinan** [comment]
I tried adding this to style.scss. Because the width is one of the styles that are used for the video in both media & text block, video, and cover blocks.

```
.wp-block-latest-posts__post-full-content video {
	width: 100%;
}
```

✅ This works when the latest posts block uses the optional grid layout.
✅ It also works for the cover and video, if the latest post blocks is the default width (content width), because these videos are then limited to the width of the latest posts block.

❌ It does not work for the media & text block, the width is too large and the content part of the block is pushed down below the video.

❌  If the latest's posts block is set to wide or full width, the video is wide or full, and does not respect the width of the video that is set in the post content of the listed post.

**Comment by carolinan** [comment]
I think there is a bigger issue with CSS for the inner blocks not loading when they are nested inside the latest posts block.

**Comment by renatho** [comment]
> I think there is a bigger issue with CSS for the inner blocks not loading when they are nested inside the latest posts block.

Yes! I did a quick exploration, and it seems the blocks are just not parsed inside the Latest Posts block. In the frontend and in the editor. I added some more details [here](https://github.com/WordPress/gutenberg/issues/69517) (closed as duplicate of this one).
